app/helpers/physical_server_helper/textual_summary.rb

```diff
@@ -60,6 +60,15 @@ def textual_ext_management_system
     textual_link(ExtManagementSystem.find(@record.ems_id))
   end
 
+  def textual_physical_switches
+    physical_switches_count = @record.physical_switches.count
+    physical_switches = {:label => _("Physical Switches"), :value => physical_switches_count, :icon => PhysicalSwitchDecorator.fonticon}
```

Comment:
You should not call the decorator directly.
```suggestion
    physical_switches = {:label => _("Physical Switches"), :value => physical_switches_count, :icon => PhysicalSwitch.fonticon}
```

Reply:
@skateman I tried this change, but I get an error if I call _fonticon_ from _PhysicalSwitch_:
```
FATAL -- : Error caught: [ActionView::Template::Error] undefined method `fonticon' for #<Class:0x00558495908390>
```

Reply:
That shouldn't happen, I'll take a look...

Reply:
Oh yes, because I need to think before I write something :confused: :wink: 
```suggestion
    physical_switches = {:label => _("Physical Switches"), :value => physical_switches_count, :icon => PhysicalSwitch.decorate.fonticon}
```

Reply:
@skateman Thanks that worked